src/Nuget/Microsoft.VisualStudio.AppDesigner/Microsoft.VisualStudio.AppDesigner.Nuspec

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.VisualStudio.AppDesigner</id>
+    <summary>AppDesigner designer host implementation for Visual Studio project property pages</summary>
+    <description>
+      This package implements the AppDesigner, which is the designer host used for project property pages among other things in Visual Studio
+    </description>
+    <language>en-US</language>
+    <version>15.0.0-alpha</version> <!-- This value will be overriden by the Nuget pack call. Having this value to satisfy the schema. -->
+    <authors>dotnet</authors>
+    <projectUrl>https://github.com/dotnet/roslyn-project-system/</projectUrl>
+    <dependencies>
+      <dependency id="Microsoft.VisualStudio.Designer.Interfaces" version="1.1.4322" />
```

Comment:
@davkean I have added all the Nuget Dependencies that were listed in the project dependencies. What would be the right approach to add the appropriate ones and remove the unnecessary ones?